theme/ckeditor5-heading/heading.css

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+.ck-heading_heading1 {
+	font-size: 1.5em;
```

Comment:
What do you think to use here variable `(--ck-font-size-normal: 1em;)` and make it:
```css
font-size: var(--ck-font-size-normal) * 1.5
```
and apply it to heading2 and heading3 too?